src/datasets/demo_only.py

```diff
@@ -71,14 +98,46 @@ def create_demo_data(env: BaseEnv, train_tasks: List[Task],
         # To prevent cheating by option learning approaches, remove all oracle
         # options from the trajectory actions, unless the options are known
         # (via CFG.included_options or CFG.option_learner = 'no_learning').
-        for act in traj.actions:
-            if act.get_option().parent not in known_options:
-                assert CFG.option_learner != "no_learning"
-                act.unset_option()
+        if CFG.demonstrator == "oracle":
+            for act in traj.actions:
+                if act.get_option().parent not in known_options:
+                    assert CFG.option_learner != "no_learning"
+                    act.unset_option()
         trajectories.append(traj)
         if CFG.make_demo_videos:
             assert monitor is not None
             video = monitor.get_video()
             outfile = f"{CFG.env}__{CFG.seed}__demo__task{idx}.mp4"
             utils.save_video(outfile, video)
     return Dataset(trajectories)
+
+
+def _human_demonstrator_policy(env: BaseEnv, idx: int, num_tasks: int,
+                               task: Task, event_to_action: Callable[
+                                   [State, matplotlib.backend_bases.Event],
+                                   Action],
+                               state: State) -> Action:  # pragma: no cover
+    # Change the backend to one that supports a GUI.
```

Comment:
could this be moved to be with the other matplotlib settings change?

Reply:
nope, it has to happen right before we call render(), and we revert this change back at the end of this function. this is because we don't want the rest of the codebase to use GUI-based matplotlib. i'll clarify the comment